spa.json

```diff
@@ -0,0 +1,13 @@
+{
+    "loadButtonLabel": "Haga clic para\nCargar\nPanorama",
+    "loadingLabel": "Cargando...",
+    "bylineLabel": "by %s",    
```

Comment:
shouldnt this be "por %s" or "de %s" ?

this is a string that tells you about the author of this picture. e.g. "mount everest by daniel"

Reply:
Fixed and I changed the filename, too